src/storage/core.rs

```diff
@@ -1220,3 +1234,71 @@ where
         active + closed
     }
 }
+
+/// Key for demonstration purposes
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct UnitKey(Vec<u8>);
```

Comment:
Is it possible to make it private (without `pub`)?

Reply:
No. Code examples must use public entities from crate